### PR TITLE
style: re-design homepage latest episodes

### PR DESCRIPTION
**Pull request by tokitouq**

### CODE REVIEW

File: src/routes/[...lang=locale]/anime/+page.svelte [hunk 1]
```diff
@@ -128,14 +123,83 @@
...
+			</div>
+		</div>
+	{/if}		
+	<!-- use Image component for just to get color -->
+	<Image
+		src={episode.cover}
+		class="hidden"
+		bind:color_palette={episode.color_palette}
+	/>
+{/snippet}
```
[the remark this answers is not in the document]
Comment (tokitouq) [reply src/routes/[...lang=locale]/anime/+page.svelte]: Resolved on https://github.com/baseplate-admin/coreproject-v4-ui/pull/73/commits/75ec40ba3d169ccdb22cb4602d5e84693759f015

File: src/routes/[...lang=locale]/anime/+page.svelte [hunk 2]
```diff
@@ -299,123 +298,71 @@
...
-						>
-							<Settings class="text-surface-900 w-[1.25vw]" />
-						</button>
-					</div>
+		<div class="flex flex-col md:gap-[1vw]">
+			<span class="md:text-[1.35vw] text-accent font-bold">{$t("home.latest_episodes.title")}</span>
+			<div class="flex md:gap-[2vw] size-full">
+				<div class="w-full grid grid-cols-2 md:gap-[1.25vw]">
+					{#each latest_episodes as episode}
+						{#if episode.color_palette.length}
```
Comment (baseplate-admin) [inline src/routes/[...lang=locale]/anime/+page.svelte:306]: {#if episode.color_palette.length}?
						
						Why is this looking like this?

File: src/routes/[...lang=locale]/anime/+page.svelte [hunk 3]
```diff
@@ -303,39 +303,37 @@
...
-										<div class="md:leading-none flex items-center md:gap-[0.5vw] md:text-[0.8vw] font-semibold">
-											<span>Ep{episode.ep_number}</span>
-											<Circle class="md:size-[0.25vw] opacity-75" />
-											<span>{episode.timestamp}</span>
-										</div>
+						<div
+							class="w-full md:h-[5vw] bg-cover bg-center relative md:rounded-[0.75vw] border md:border-[0.15vw]"
+							style="
+								background-image: url({episode.banner});
+								border-color: #{episode.color_palette.length && rgbHex(...episode.color_palette[0])};
```
Comment (baseplate-admin) [inline src/routes/[...lang=locale]/anime/+page.svelte:326]: Change this 

```css
								border-color: #{episode.color_palette.length && rgbHex(...episode.color_palette[0])};```

**Review by baseplate-admin — CHANGES_REQUESTED** [review]

File: src/routes/[...lang=locale]/anime/+page.svelte (hunk 3 above)
Comment (tokitouq) [reply src/routes/[...lang=locale]/anime/+page.svelte]: Resolved on https://github.com/baseplate-admin/coreproject-v4-ui/pull/73/commits/02b3449038a5dcee5a8922ec9a27216f032b24ec

**Pull request merged**